Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM node:10-alpine
+FROM node:12-alpine
 
 RUN apk add git
 
 COPY . /
 
-RUN npm install --production
+RUN yarn
```

Comment:
FFTI: might be good to keep the `--production` to keep CI fast by skipping devDependencies